setup.py

```diff
@@ -1,20 +1,31 @@
 from setuptools import setup, find_packages
 
 import os
+
+
+with open("README.rst", "r", encoding="utf-8") as fh:
+    long_description = fh.read()
+
+
 def package_files(directory):
     paths = []
     for (path, directories, filenames) in os.walk(directory):
         for filename in filenames:
             paths.append(os.path.join('..', path, filename))
     return paths
 
-extra_files = package_files('ecldoc/Templates')
+
+extra_files = package_files('src/ecldoc/Templates')
```

Comment:
What's this change for?

Reply:
because setup.py file moved to root so we need to make these changes